standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/cache/CacheUtils.java

```diff
@@ -144,4 +147,15 @@ public static boolean matches(String name, String pattern) {
     }
     return false;
   }
+
+  public static boolean validateShouldCacheTableCanUseEventIsActiveTransaction(String catName, String dbName,
```

Comment:
It is not used anywhere other than below method. We can remove it and move these validations there.

Reply:
removed